portfolio/src/main/java/com/google/sps/data/Information.java

```diff
@@ -0,0 +1,85 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Information {
+    private long id;
+    private String email;
+    private ArrayList<String> type;
+    //private String type;
+    private String name;
+    private String comment;
+    private long timestamp;
+  
+  public Information()
+  {
+
+  }
+
+   public Information(long id, String email, ArrayList<String> type, String name, String comment, long timestamp)
```

Comment:
Possibly something we could do is use the builder format for our class since there are so many parameters in our constructor. I can try to help with that later on